cmd/sandbox/sandbox.go

```diff
@@ -0,0 +1,39 @@
+package sandbox
+
+import (
+	cmdcore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/spf13/cobra"
+)
+
+// Long descriptions are whitespace sensitive when generating docs using sphinx.
+const (
+	sandboxShort = `Used for playing with sandbox.`
+	sandboxLong  = `
+Example Create sandbox cluster.
+::
+
+ flytectl sandbox cluster 
```

Comment:
is this example correct, 

instead it should be 
`flytectl sandbox start`

`flytectl sandbox teardown`